tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -356,8 +356,11 @@ Then('the last uploaded folder should be listed on the webUI', function () {
     .waitForFileVisible(folder)
 })
 
-Then('file/folder {string} should not be listed on the webUI', function (folder) {
-  return client.page.FilesPageElement.filesList().assertElementNotListed(folder)
+Then('file/folder {string} should not be listed on the webUI', async function (folder) {
+  const state = await client.page.FilesPageElement.filesList().isElementListed(folder)
+  return client.assert.ok(
```

Comment:
you could use `strictEqual` https://nodejs.org/api/assert.html#assert_assert_strictequal_actual_expected_message

also in other places

Reply:
If we are testing whether the value is `truthy` or `falsy` usually `assert.ok` is preferred because there is no need of `expected-param` on such assertions!
[https://nodejs.org/api/assert.html#assert_assert_ok_value_message](url)